modules/core/src/passes/layers-pass.ts

```diff
@@ -288,7 +298,10 @@ export default class LayersPass extends Pass {
     layerIndex: number,
     viewport: Viewport
   ): LayerParameters {
-    return layer.props.parameters;
+    const {getPolygonOffset} = layer.props;
```

Comment:
I wonder if we should have deprecated `getPolygonOffset` in v9. It is strange that it is called and accessor in the docs, but is in fact a roundabout way of setting GPU parameters.

Is there any reason to keep the function? I think it would be clearer to pass `depthBias(SlopeScale)` in `props.parameters`. In addition if `depthBias` is passed in `props.parameters` it will be overwritten even if `getPolygonOffset` isn't specified.

Reply:
I certainly agree that it should not be called an "accessor" if that is really the case, as accessors operate on data rows and this function operates on a layer. 

We can still deprecate a function at any time. But I believe the functionality of being able to adjust depth bias based on layer index is needed to address Z fighting so we can't just drop it, we need to have a replacement for this function.